src/styles/index.scss

```diff
@@ -1,3 +1,109 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
```

Comment:
Move to the card file
<img width="516" alt="image" src="https://github.com/user-attachments/assets/f4629455-aebe-4560-a265-4acef606e71d">